docs/src/tutorials/pbt-and-tla.md

```diff
@@ -0,0 +1,1000 @@
+# Tutorial on Checking ERC20 with Property-Based Testing and TLA+
```

Comment:
Can we add a TOC to individual pages? This is quite a long document, and I would've liked to locate subsections.